### ISSUE TITLE
LegacyMappingTableCtoD 2525DeltaName column - many entries have ": Unknown"

### ISSUE THREAD

**Issue by csmoore** [Esri/joint-military-symbology-xml#198]
Many of the entries in the "2525DeltaName" column of the [LegacyMappingTableCtoD ] 
(https://github.com/Esri/joint-military-symbology-xml/blob/master/samples/legacy_support/LegacyMappingTableCtoD.csv
) have extra/superfluous ": Unknown" appended to the name

Ex: https://github.com/Esri/joint-military-symbology-xml/blob/master/samples/legacy_support/LegacyMappingTableCtoD.csv#L112 

The ": Unknown" should be removed prior to export if possible

**Comment by joebayles** [comment]
@abouffard can you update this as necessary?

**Comment by joebayles** [comment]
@csmoore agreed, do you know if this is updated automatically through a script? If not, I can just do it manually.

**Comment by csmoore** [comment]
@joebayles - I believe all of the csv's are created through the [export C# app](https://github.com/Esri/joint-military-symbology-xml/tree/dev/source/JointMilitarySymbologyLibraryCS/jmsml)

**Comment by abouffard** [comment]
@joebayles @csmoore - yes,that CSV is updated using the jmsml console app, by running https://github.com/Esri/joint-military-symbology-xml/blob/dev/samples/legacy_support/exportLegacyData.bat

**Comment by joebayles** [comment]
So, could we write an exception in that bat file to disregard "Unknown"?

**Comment by abouffard** [comment]
No.  You don't want to disregard the lines containing the Unknowns, just remove the " : Unknown".

Its been a while but I believe the "Unknowns" are appearing on the end of full frame entity names because JMSML uniquely identifies four different shapes of full frame entities by appending the standard identity string on the end of the basic name, and opts to write out the "Unknown" name to this CSV.

You'd either have to change the logic in the JMSML ETL.cpp code to remove the word Unknown from those strings, prior to output, or write a RegEx/Perl post process that removes the extra " : Unknown" sub strings from the ends of the names in the CSV.
